cmd/frontend/graphqlbackend/textsearch.go

```
@@ -554,9 +558,13 @@ func searchFilesInRepos(ctx context.Context, args *search.TextParameters, stream
 					partition := make(map[string][]string)
 					var repos []*search.RepositoryRevisions
 
-					for _, m := range event.results {
-						name := string(m.Repo.Name())
-						partition[name] = append(partition[name], m.JPath)
+					for _, m := range event.Results {
+						fm, ok := m.ToFileMatch()
+						if !ok {
+							panic("this should always be a FileMath")
```

can we return an error here instead?

---

done